frequently-asked-questions.md converted into YML

v-hearya opened the pull request:
Per the [this page](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-faq-guide?branch=master), Updated following Markdown to YAML template, to improve the discoverability of the FAQ pages in Google.

v-hearya committed:
frequently-asked-questions.md converted into YML

Youssef1313 commented on docs/framework/data/adonet/sql/linq/frequently-asked-questions.yml:
```diff
@@ -0,0 +1,253 @@
+  dev_langs:
+    - "csharp"
+    - "vb"
+  ms.assetid: 252ed666-0679-4eea-b71b-2f14117ef443
```
I think ms.assetid isn't used.

```suggestion
```

IEvangelist commented:
Attention: @Youssef1313 -- seeing _build verifier_ failure here, not sure why?

IEvangelist commented on docs/framework/data/adonet/sql/linq/frequently-asked-questions.yml:
```diff
@@ -0,0 +1,253 @@
+      - question: |
+          How long does my database connection remain open?
+        answer: |
+          A connection typically remains open until you consume the query results. If you expect to take time to process all the results and are not opposed to caching the results, apply <xref:System.Linq.Enumerable.ToList%2A> to the query. In common scenarios where each object is processed only one time, the streaming model is superior in both `DataReader` and [!INCLUDE[vbtecdlinq](../../../../. ...
```
This `[!INCLUDE]` is not rendering correctly, I do not believe the that YML supports the include syntax like MD files do.

v-hearya replied:
Under Question [!INCLUDE] doesn't support. We can use it in Answer or summary.

Youssef1313 commented:
@IEvangelist You can safely ignore this failure for now (the build won't fail on subsequent PRs). This is a case where a deleted file doesn't need a redirection that I'm not handling. I opened https://github.com/dotnet/docs-actions/issues/12 to track fixing this.

IEvangelist commented on docs/framework/data/adonet/sql/linq/frequently-asked-questions.yml:
```diff
@@ -0,0 +1,253 @@
+  - name: SQL-Injection Attacks
+    questions:
+      - question: |
+          How is [!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)] protected from SQL-injection attacks?
```
```suggestion
          How is LINQ to SQL protected from SQL-injection attacks?
```

We should replace ALL occurrences of `[!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)]` with `LINQ to SQL`.

IEvangelist reviewed (changes requested):
Some issues with the includes...

![image](https://user-images.githubusercontent.com/7679720/123658165-168a7a00-d7f7-11eb-9b1f-e43b544c343e.png)


![image](https://user-images.githubusercontent.com/7679720/123652461-eee4e300-d7f1-11eb-9a23-b5d70cb7b819.png)

v-hearya committed:
Update docs/framework/data/adonet/sql/linq/frequently-asked-questions.yml

Co-authored-by: Youssef Victor <youssefvictor00@gmail.com>

v-hearya committed:
Updated questions

[!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)] replaced with LINQ to SQ in Questions only as they don't support [!INCLUDE].

v-hearya commented:
@IEvangelist - Neede changes made, please verify.

v-hearya committed:
link tag removed from Questions as they don't support it
